applications/resturant/apps.py

```
@@ -1,9 +1,11 @@
 from django.apps import AppConfig
 
+from utils.logging import setup_loguru
```

Pylint (3.3.7)

[error] Unable to import 'utils.logging'

(E0401)